IPIP/0001-lightweight-improvement-proposal-process.md

```diff
@@ -54,29 +59,33 @@ document** based on the template in [`ipfs/specs/IPIP/0000-template.md`](./0000-
 When a new specification file is added to the repo, it should be based on
 the template at [`ipfs/specs/template.md`](../template.md).
 
-### Reviewing IPIPs
-
-[Specs Stewards] will review new IPIP PRs during periodical (best-effort) triage.
+When naming a new proposal, don't try to introduce an IPIP number; we will do that only for
+IPIPs that are approved before accepting into master.
 
-IPFS Community is encouraged to participate in the review process.
+Proposals are officially submitted when a pull request into master is opened
 
-IPIP can be either:
-- merged,
-- rejected (PR close without merging),
-- deferred (converting PR back to a draft).
+Proposals that were reviewed and rejected will be moved into /archive folder and then merged into master
 
-The final decision belongs to [Specs Stewards].
-
-### Merging IPIPs
-
-PR with a IPIP can be merged only after two [Specs Stewards] approve it and
-there are no objections from other Stewards.
+### Reviewing IPIPs
 
-IPIP number is assigned before the PR merge.
+1. [Specs Stewards] will do an initial triage of newly opened PRs roughly monthly. They'll try to filter out
+noise, so that community consideration is given only to reasonable proposals; others they'll reject.
+2. Specs Stewards will post to the forums linking to the proposal; directing feedback/discussion to
+take place in Github on the PR
+3. After a month of discussion, Specs Stewards will review again. If there are no substantive disagreements
+with the proposal, including within Spec Stewards, the proposal will be approved.
+4. If discussion or modification is still underway and appears to be leading to a resolution, it can be held
+open for another month
+5. Proposals that are generating ongoing discussion and seem contentious or stuck will be brought in for
+consideration at a monthly sync, to be announced at least a week ahead of time on the forum.
+6. After discussion, Spec Stewards will make call on whether to approve or reject the proposal.
+7. At this point approved proposals get assigned a number (encoded in the filename),
+and merged into the IPIP folder on master. Rejected
+proposals should be also merged to master, but in a subfolder called "Rejected". Proposals rejected in intial
+triage will simply have the PR declined.
+8. IPIP author and two approving [Specs Stewards] are added to `CODEOWNERS` fileto be
+automatically asked to review any future changes to files added or modified by the IPIP.
```

Comment:
```suggestion
8. IPIP author and two approving [Specs Stewards] are added to `CODEOWNERS` file to be
automatically asked to review any future changes to files added or modified by the IPIP.
```

Reply:
I agree with this - I think documenting the process itself in an IPIP is inconsistent with the notion of IPIPs documenting changes. I'll make this modification.